### ISSUE TITLE
[Bug]: navigation icon hover states are no longer circular

### ISSUE THREAD

**Issue by kgryte** [stdlib-js/www#44]
### Description

Encountered an error when hovering over top-navigation icons. They are oblong, rather than circular. This change likely happened during the migration from Material UI to Mui v5.

### Related Issues

None.

### Questions

No.

### Reproduction

```shell
-  hover over top-navigation icons.
```

### Expected Results

```shell
The hover backgrounds should be circular.
```

### Actual Results

```shell
The hover backgrounds are oblong.
```

### Environments

Firefox, Chrome, Safari, Microsoft Edge, Internet Explorer, Other Browser

### Checklist

- [X] Read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.

**Comment by Planeshifter** [comment]
This should be resolved by the CSS changes in 30f48188c9fe8e85d68a5b804951f70000156b52.

**Comment by kgryte** [comment]
I believe this has been resolved. We no longer have circular hover states. Closing...
